app/utility/base_knowledge_svc.py

```diff
@@ -283,8 +284,9 @@ def _destroy():
         tarball_path = os.path.join(DATA_BACKUP_DIR, f'backup-{timestamp}.tar.gz')
 
         with tarfile.open(tarball_path, 'w:gz') as tarball:
-            tarball.add(FACT_STORE_PATH)
-            BaseKnowledgeService._delete_file(FACT_STORE_PATH)
+            if os.path.isfile(FACT_STORE_PATH):
```

Comment:
No comment on changed code, but what about the docstrings for this method? Seem off.

Reply:
They're a different style than normal - a side effect of how I originally wrote the documentation and distinguished between core and support functions. I'll update it to be more in line with everything else.